lambda/handlers/intentHandler.js

```
@@ -191,23 +618,31 @@ const NextPrayerTimeIntentHandler = {
             mosqueTimesData.aidPrayerTime2,
           ];
           // Find the first non-null Eid time
-          const firstNonNullEid = eidTimes.filter((time) => time !== null && time !== undefined).join(", ");
+          const firstNonNullEid = eidTimes
+            .filter((time) => time !== null && time !== undefined)
+            .join(", ");
```

Biome (2.4.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)